docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -18,6 +18,14 @@ default:: `alpha`
 
 The channel from where the operator should be fetched.
 
+== `openshift_version`
+
+[horizontal]
+type:: string
+default:: `4.9`
+
+The openshift version which will be used as the image tag in KeepalivedGroup CRD.
```

Comment:
This is not entirely true, since it gets appended to `v`.

```suggestion
The OpenShift version used to set the corresesponding image tag for the KeepalivedGroup(s).
```

Reply:
yep, missed that, comited your suggestion